pkg/cluster/deploystorage.go

```diff
@@ -72,9 +72,7 @@ func (m *manager) ensureResourceGroup(ctx context.Context) error {
 		ManagedBy: to.StringPtr(m.doc.OpenShiftCluster.ID),
 	}
 	if m.env.IsLocalDevelopmentMode() {
-		// TODO: ideally this should be removed, but removing it causes the
-		// purge script to ignore RGs in CI E2E.  Fix that at the same time.
-		group.ManagedBy = nil
+		group.Tags = map[string]*string{"purge": to.StringPtr("true")}
```

Comment:
You will want to update `group.Tags` if it's not `nil` as the resource group could have tags set on it already for other reasons (`persist` tag to not delete the resource group, etc.).  

`ensureResourceGroup` runs on `adminUpdate` so it can overwrite tags if a dev runs it after setting a tag.

Reply:
done

Reply:
I misspoke before.  

What's the behavior if you run a `adminUpdate` on a development cluster with tags already?  Since you're constructing the resourceGroup variable and setting tags on it, will it wipe out the preexisting tags?  

If not, noop here.  Sorry about misdirection.